tailwind.config.js

```diff
@@ -72,6 +72,7 @@ module.exports = {
         4.5: "1.1875rem",
         5.1: "1.3125rem",
         6.25: "1.5625rem",
+        6.5 : "1.6875rem",
```

Comment:
To get the value to use as tailwind key we take the value in pixels divide by 4. for your case is `` 27px / 4 = 6.75``